ogc/style-wms-layer/src/main/java/com/esri/samples/style_wms_layer/StyleWmsLayerSample.java

```diff
@@ -55,8 +55,10 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create a map and add it to the map view
-      ArcGISMap map = new ArcGISMap(Basemap.createImagery());
+      // create a map with spatial reference appropriate for the service
+      ArcGISMap map = new ArcGISMap(SpatialReference.create(26915));
+      map.setMinScale(7000000.0);
```

Comment:
let's drop the decimal and keep it as a whole number (in keeping with other setting scales in the repo)
```suggestion
      map.setMinScale(7000000);
```